package.json

```diff
@@ -32,20 +32,9 @@
     "babel-polyfill": "^6.3.14"
   },
   "devDependencies": {
-    "babel-cli": "^6.4.0",
```

Comment:
All this dependencies need for local testing&publishing
Also, when this package published, not need build it with babel6 and babel7 - enough one build with any version

Maybe change before_install to after_install in travis.yaml?
Then travis may test both configuration, but at local development/testing/publishing may use babel6 (or babel7)

Reply:
👍 Sure, that makes sense, will fix tomorrow!